### ISSUE TITLE
Website blocked as malware

### ISSUE THREAD

**Issue by kelle** [SIMPLE-AstroDB/SIMPLE-web#100]
My college is blocking the website as by "CUNY Internet Malware Protection Services".  I've contacted the helpdesk to see if they can tell me why it's being blocked so we can fix it.

> Web Page Blocked by CUNY Internet Malware Protection Services
> Access to the web page you were trying to visit has been blocked because it contains malware or related unsafe content. Please contact your campus IT Help Desk if you believe this determination is in error.
> 
> Help Desk contact information can be found at http://www2.cuny.edu/about/university-resources/tech-resources-help-desks/
> 
> User / Originating IP Address: 146.95.157.37
> URL: simple-bd-archive.org/
> Category: parked

**Comment by Will-Cooper** [comment]
Hmm curious, it's fine on my end; is it perhaps because it's an http link?

**Comment by kelle** [comment]
Update from Hunter helpdesk: 

> The reported site ([simple-bd-archive.org](http://simple-bd-archive.org/)) is currently blocked, but not directly by our system, but through a block-list subscription service. I can request a status change...

**Comment by kelle** [comment]
I can access it now! Woot!

> The site has been categorized as safe by the referenced service. It will take a few hours for the change to go into effect, but unblock for all subscribers of said list (global in scope). 
> 
> There are many potential reasons a given site winds up on a block list. It is possible, the site was mistakenly reported as malicious, or the URL was used as a spoofed sending address in a spam campaign. Spam and other blocking services generally will not divulge their methods in order to maintain their own security.
